packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -381,11 +381,13 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
           </Stack.Item>
           {
             /* @conditional-compile-remove(video-background-effects) */
-            <VideoEffectsPane
-              showVideoEffectsOptions={showVideoEffectsPane}
-              setshowVideoEffectsOptions={setShowVideoEffectsPane}
-              adapter={callAdapter}
-            />
+            <CallAdapterProvider adapter={callAdapter}>
+              <VideoEffectsPane
+                showVideoEffectsOptions={showVideoEffectsPane}
+                setshowVideoEffectsOptions={setShowVideoEffectsPane}
+                adapter={callAdapter}
```

Comment:
if we are not wrapping this in a provider do we need to still be passing it as a prop to the pane?